variants/arduino_nano_nora/pins_arduino.h

```diff
@@ -0,0 +1,125 @@
+#ifndef Pins_Arduino_h
+#define Pins_Arduino_h
+
+#include <stdint.h>
+#include "soc/soc_caps.h"
+
+#define USB_VID 0x2341
+#define USB_PID 0x0070
+
+#ifndef __cplusplus
+#define constexpr const
+#endif
+
+// primary pin names
+
+#if defined(BOARD_HAS_PIN_REMAP) && !defined(BOARD_USES_HW_GPIO_NUMBERS)
+
+// Arduino style definitions (API uses Dx)
+
+#define NUM_DIGITAL_PINS        25                    // 25 I/O lines exported
+#define NUM_ANALOG_INPUTS       20                    // 20 CPU ADC inputs, not all exported
+#define EXTERNAL_NUM_INTERRUPTS NUM_DIGITAL_PINS      // All GPIOs
+
+#define analogInputToDigitalPin(p)  (((p)<NUM_ANALOG_INPUTS)?(analogChannelToDigitalPin(p)):-1)
+#define digitalPinToInterrupt(p)    (((p)<NUM_DIGITAL_PINS)?(p):NOT_AN_INTERRUPT)
+#define digitalPinHasPWM(p)         (p < NUM_DIGITAL_PINS)
+
+static constexpr uint8_t D0         = 0; // also RX
+static constexpr uint8_t D1         = 1; // also TX
+static constexpr uint8_t D2         = 2;
+static constexpr uint8_t D3         = 3; // also CTS
+static constexpr uint8_t D4         = 4; // also DSR
+static constexpr uint8_t D5         = 5;
+static constexpr uint8_t D6         = 6;
+static constexpr uint8_t D7         = 7;
+static constexpr uint8_t D8         = 8;
+static constexpr uint8_t D9         = 9;
+static constexpr uint8_t D10        = 10; // also SS
+static constexpr uint8_t D11        = 11; // also MOSI
+static constexpr uint8_t D12        = 12; // also MISO
+static constexpr uint8_t D13        = 13; // also SCK, LED_BUILTIN
+static constexpr uint8_t LED_RED    = 14;
+static constexpr uint8_t LED_GREEN  = 15;
+static constexpr uint8_t LED_BLUE   = 16; // also RTS
+
+static constexpr uint8_t A0         = 17; // also DTR
+static constexpr uint8_t A1         = 18;
+static constexpr uint8_t A2         = 19;
+static constexpr uint8_t A3         = 20;
+static constexpr uint8_t A4         = 21; // also SDA
+static constexpr uint8_t A5         = 22; // also SCL
+static constexpr uint8_t A6         = 23;
+static constexpr uint8_t A7         = 24;
+
+#else
+
+// ESP32-style definitions (API uses GPIOx)
+
+#define NUM_DIGITAL_PINS        SOC_GPIO_PIN_COUNT    // GPIO 0..48, not all exported
+#define NUM_ANALOG_INPUTS       20                    // GPIO 1..20, not all exported
+#define EXTERNAL_NUM_INTERRUPTS NUM_DIGITAL_PINS      // All GPIOs
+
+#define analogInputToDigitalPin(p)  (((p)<NUM_ANALOG_INPUTS)?(analogChannelToDigitalPin(p)):-1)
+#define digitalPinToInterrupt(p)    (((p)<NUM_DIGITAL_PINS)?(p):NOT_AN_INTERRUPT)
+#define digitalPinHasPWM(p)         (p < NUM_DIGITAL_PINS)
```

Comment:
This part is no longer needed to be defined in the pins_arduino.h file for each variant.
It is defined globally for each SoC. Please remove it to prevent warnings.